scss/_toasts.scss

```diff
@@ -1,4 +1,5 @@
 .toast {
+  flex-basis: $toast-max-width; // prevents from shrinking in IE11, when in a flex container https://github.com/twbs/bootstrap/issues/28341
```

Comment:
Nitpicking can you move the comment above the property so that it's not so long? :)